### ISSUE TITLE
Chatroom edit lost in UI upgrade?

### ISSUE THREAD

**Issue by HaakonME** [redmatrix/hubzilla#727]
It would be nice if it was possible to edit chatrooms after creation, especially the ACL to be able to include new Zot Access Tokens.

**Comment by git-marijus** [comment]
I don't think we ever had the possibility to edit chatrooms. It is desired though...
